values.yaml

```diff
@@ -790,6 +790,36 @@ concourse:
       ##
       useCaCert: false
 
+    opa:
+      ## Open Policy Agent policy check endpoint.
+      ##
+      ## Example: https://opa.example.com:8181/v1/data/concourse/allow
+      ##
+      url:
+
+      ## Open Policy Agent API request timeout.
+      ##
+      timeout:
```

Comment:
Can we add an example of what format the timeout value is expected to be in. Is it always seconds? Can I do `30s` or `200ms`?